src/java/main/org/apache/zookeeper/server/quorum/LearnerHandler.java

```diff
@@ -51,6 +53,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.net.ssl.SSLSocket;
+
```

Comment:
Looks like these imports are not used. Another look is this file can also be removed from change list because nothing changed.